<p><link href="style.css" rel="stylesheet"></link></p>

<h2>Ideas from ISMB 2012</h2>

<h5>Finding subpopulations in cell lines/tumors, perhaps using copy number data?</h5>

<p>Using LOH or loss from the Illumina 2.5M array we may define regions of LOH and copy number loss. These should have homozygous mutations only. Any heterozygous mutations in these regions may point to subclonal populations.</p>

<p>This is currently investigated by the BC cancer people. Maybe we can also do this for our clinical samples</p>

<p>Another idea might be to mix different cell lines to create a synthetic dataset of subclonal populations and see how we can detect the mixing coefficient using genomic features. </p>

<p>Determination of a specific subclonal population is very significant, as they might give rise to resistance. For example, a sub population of met amplified cells gives rise to EGFR resistance. Looking in the met locus, if a region of different copy number can actually be found, the allele frequency might give hints toward subclonality.</p>

<h5>Advantages of whole genome sequencing</h5>

<p>Many people are ignoring whole genome genome data in favor of Exome sequencing. We should try to leverage the advantages of WGS:</p>

<ul>
<li>Accurate SV detection</li>
<li>Sequence at the breakpoints, pointing towards repair pathways </li>
</ul>

<h5>Current downsides of sequencing</h5>

<p>Mapping causes a lot of errors, and restricts our efforts to detect variants in all their forms. Filtering can get you some way, but it could be better restart and approach the problem from a different angle.</p>
